core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -0,0 +1,1121 @@
+fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Action): Color {
+    return when (token) {
+        OudsColorKeyToken.Action.Disabled -> actionColors.disabled
+        OudsColorKeyToken.Action.DisabledOnBgEmphasized -> actionColors.disabledOnBgEmphasized
+        OudsColorKeyToken.Action.NegativeEnabled -> actionColors.negativeEnabled
+        OudsColorKeyToken.Action.NegativeFocus -> actionColors.negativeFocus
+        OudsColorKeyToken.Action.NegativeHover -> actionColors.negativeHover
+        OudsColorKeyToken.Action.NegativeLoading -> actionColors.negativeLoading
+        OudsColorKeyToken.Action.NegativePressed -> actionColors.negativePressed
+        OudsColorKeyToken.Action.PrimaryEnabled -> actionColors.primaryEnabled
+        OudsColorKeyToken.Action.PrimaryEnabledOnBgEmphasized -> actionColors.primaryEnabledOnBgEmphasized
+        OudsColorKeyToken.Action.PrimaryFocus -> actionColors.primaryFocus
+        OudsColorKeyToken.Action.PrimaryFocusOnBgEmphasized -> actionColors.primaryFocusOnBgEmphasized
+        OudsColorKeyToken.Action.PrimaryHover -> actionColors.primaryHover
+        OudsColorKeyToken.Action.PrimaryHoverOnBgEmphasized -> actionColors.primaryHoverOnBgEmphasized
+        OudsColorKeyToken.Action.PrimaryLoading -> actionColors.primaryLoading
+        OudsColorKeyToken.Action.PrimaryLoadingOnBgEmphasized -> actionColors.primaryLoadingOnBgEmphasized
+        OudsColorKeyToken.Action.PrimaryPressed -> actionColors.primaryPressed
+        OudsColorKeyToken.Action.PrimaryPressedOnBgEmphasized -> actionColors.primaryPressedOnBgEmphasized
+        OudsColorKeyToken.Action.SecondaryEnabled -> actionColors.secondaryEnabled
+        OudsColorKeyToken.Action.SecondaryEnabledOnBgEmphasized -> actionColors.secondaryEnabledOnBgEmphasized
+        OudsColorKeyToken.Action.SecondaryFocus -> actionColors.secondaryFocus
+        OudsColorKeyToken.Action.SecondaryFocusOnBgEmphasized -> actionColors.secondaryFocusOnBgEmphasized
+        OudsColorKeyToken.Action.SecondaryHover -> actionColors.secondaryHover
+        OudsColorKeyToken.Action.SecondaryHoverOnBgEmphasized -> actionColors.secondaryHoverOnBgEmphasized
+        OudsColorKeyToken.Action.SecondaryLoading -> actionColors.secondaryLoading
+        OudsColorKeyToken.Action.SecondaryLoadingOnBgEmphasized -> actionColors.secondaryLoadingOnBgEmphasized
+        OudsColorKeyToken.Action.SecondaryPressed -> actionColors.secondaryPressed
+        OudsColorKeyToken.Action.SecondaryPressedOnBgEmphasized -> actionColors.secondaryPressedOnBgEmphasized
+        OudsColorKeyToken.Action.Selected -> actionColors.selected
+        OudsColorKeyToken.Action.SelectedOnBgEmphasized -> actionColors.selectedOnBgEmphasized
+        OudsColorKeyToken.Action.Visited -> actionColors.visited
+        OudsColorKeyToken.Action.VisitedOnBgEmphasized -> actionColors.visitedOnBgEmphasized
+    }
+}
+
+@Stable
+fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Always): Color {
+    return when (token) {
```

Comment:
We can use `with(alwaysColors)` to simplify the following lines.

Same comment for other `fromToken` methods.